changes/ocd-3934.sql

```diff
@@ -0,0 +1,16 @@
+-- SCHEMA: shared_data
+DROP SCHEMA IF EXISTS shared_data;
+
+CREATE SCHEMA IF NOT EXISTS shared_store
+    AUTHORIZATION openchpl_dev;
+
+GRANT ALL ON SCHEMA shared_store TO openchpl_dev;
```

Comment:
Do we need these two GRANT commands knowing that the grant-all script will get run at the end of the load-pending-changes script?